docs/guides/governance.md

```diff
@@ -52,7 +52,7 @@ Where `proposal.json` is:
 ```json
 {
   "type": "Text",
-  "title": "My Cool Proposal",
+  "title": "Community Pool Spend: Super important committee",
```

Comment:
Now it says `Community Pool Spend` on a text prop. Is this on purpose?

Reply:
No sorry, fixed. Thanks Tom